### PR TITLE
Only add clearfix on block list widget

### PR DESCRIPTION
**Pull request by nymkappa**
Fixes https://github.com/mempool/mempool/issues/3034

### CODE REVIEW

File: frontend/src/app/components/blocks-list/blocks-list.component.html [hunk 1]
```diff
@@ -123,8 +123,8 @@ <h1 *ngIf="!widget" class="float-left" i18n="master-page.blocks">Blocks</h1>
       (pageChange)="pageChange(page)" [boundaryLinks]="true" [ellipses]="false">
     </ngb-pagination>
 
-    <div class="clearfix"></div>
-    <br>
+    <div *ngIf="!widget" class="clearfix"></div>
```
Comment (softsimon) [inline frontend/src/app/components/blocks-list/blocks-list.component.html:126]: If you wrap both elements in `<ng-template [ngIf]="!widget">` you save the logic two if statements

File: frontend/src/app/lightning/lightning-dashboard/lightning-dashboard.component.html [hunk 2]
```diff
@@ -102,7 +102,6 @@ <h5 class="card-title d-inline" i18n="lightning.connectivity-ranking">Connectivi
     <a *ngIf="!officialMempoolSpace" [routerLink]="['/tx/push' | relativeUrl]" i18n="shared.broadcast-transaction|Broadcast Transaction">Broadcast Transaction</a>
   </div>
 
-  <br>
```
Comment (softsimon) [inline frontend/src/app/lightning/lightning-dashboard/lightning-dashboard.component.html:105]: Do we want to remove the last break row on the Lightning dashboard? The other dashboards have a break row in the bottom.

Comment (nymkappa) [reply frontend/src/app/lightning/lightning-dashboard/lightning-dashboard.component.html:105]: Yes actually the lightning dashboard has two `<br>` at the bottom of the page. I removed the wrong one. Fixed.

File: frontend/src/app/components/blocks-list/blocks-list.component.html (hunk 1 above)
Comment (nymkappa) [reply frontend/src/app/components/blocks-list/blocks-list.component.html]: Fixed

File: frontend/src/app/components/blocks-list/blocks-list.component.html [hunk 3]
```diff
@@ -123,8 +123,10 @@ <h1 *ngIf="!widget" class="float-left" i18n="master-page.blocks">Blocks</h1>
       (pageChange)="pageChange(page)" [boundaryLinks]="true" [ellipses]="false">
     </ngb-pagination>
 
-    <div class="clearfix"></div>
-    <br>
+    <ng-template [ngIf]="!widget">
+      <div class="clearfix"></div>
+      <br *ngIf="!widget">
```
Comment (softsimon) [inline frontend/src/app/components/blocks-list/blocks-list.component.html:128]: This `*ngIf="!widget"` is not required.

**Review by softsimon — APPROVED** [review]
tested ACK @ v2.5.0-dev [[c1fc60e6](https://github.com/mempool/mempool/commit/c1fc60e6)]

**Pull request merged**